docs/introduction.rst

```diff
@@ -133,27 +133,17 @@ in our code repository to see how different actions can be performed.
 
 
 
-Where we stopped/What RackHD doesn’t do
+What RackHD Doesn’t Do
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We intentionally stopped at two conceptual boundaries - the first, we didn’t
-attempt to replicate all the work and effort that’s gone into software
-configuration management systems. Ansible, Puppet, Chef, and so forth have a
-long history of dealing with a lot of that complexity, and doing it pretty well.
-We made sure our workflow system could integrate seamlessly with those kinds of
-systems - making a call to register a machine with a Puppet or Chef service, or
-in the case of ansible, some example hooks for how to invoke a playbook or
-arbitrary script on the remote machine.
-
-The second - we intentionally made RackHD a comparatively passive system. You
-can embed a lot of logic in a workflow, but we stopped short of building in more
-complex logic that amounting to functions more commonly done as scheduling -
-choosing which machines to install with what OS, etc. We expect that someone,
-or some thing, will be making those relevant choices - a layer above hardware
-management and orchestration that we saw as “infrastructure orchestration and
-management”. We documented and exposed all of the events around the workflow
-engine to be utilized, extended, and even incorporated by an infrastructure
-management system - but we didn’t take RacKHD directly into that layer.
+RackHD is a comparatively passive system. Workflows do not contain the complex logic for
```

Comment:
"Workflows do not contain the complex functionality" -> "Workflows are not intended to contain the complex functionality..."